package.json

```diff
@@ -7,7 +7,7 @@
   },
   "scripts": {
     "serve": "NODE_ENV=development npm run generate:tokenlists && vue-cli-service serve",
-    "build": "npm run generate:tokenlists && vue-cli-service build",
+    "build": "npm run generate:tokenlists && npm run typecheck && vue-cli-service build",
```

Comment:
@garethfuller Do you think it's a good idea to run typecheck here, before build, or is it enough to only run it in the test job in CI?

Reply:
How long does it take?